staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -377,6 +409,21 @@ var (
 			},
 		},
 	)
+	SuggestedFlowSchemaEvents = newFlowSchema(
+		"events", SuggestedPriorityLevelConfigurationEvent.Name, 450,
+		flowcontrol.FlowDistinguisherMethodByUserType,
+		flowcontrol.PolicyRulesWithSubjects{
+			Subjects: groups(user.AllAuthenticated), // the nodes group
+			ResourceRules: []flowcontrol.ResourcePolicyRule{
+				resourceRule(
+					[]string{"create", "update", "delete"},
```

Comment:
Event client does use Patch requests for EventSeries https://github.com/kubernetes/client-go/blob/master/tools/events/event_broadcaster.go#L240

Reply:
Good catch, thanks. Updated.

Reply:
Good catch. Fixed here and in #129008